File: admoblibrary/src/main/java/com/vapp/admoblibrary/ads/AdCallBackInterLoad.java
package com.vapp.admoblibrary.ads;

import com.google.android.gms.ads.interstitial.InterstitialAd;

public interface AdCallBackInterLoad {
    void onAdClosed();
    void onEventClickAdClosed();
    void onAdShowed();
    void onAdLoaded(InterstitialAd interstitialAd, boolean isLoading);
    void onAdFail( boolean isLoading);
}

File: admoblibrary/src/main/java/com/rikatech/admoblibrary/callback/AdCallBackInterLoad.kt
package com.rikatech.admoblibrary.callback

import com.google.android.gms.ads.interstitial.InterstitialAd

interface AdCallBackInterLoad {
    fun onAdClosed()
    fun onEventClickAdClosed()
    fun onAdShowed()
    fun onAdLoaded(interstitialAd: InterstitialAd?, isLoading: Boolean)
    fun onAdFail(message: String?)
}
